lib/disc.rb

```diff
@@ -110,10 +119,14 @@ def disc_options
       end
 
       def queue
-        disc_options.fetch(:queue, 'default')
+        @queue || Disc.default_queue
```

Comment:
Heh, now there's another bug :stuck_out_tongue: 

``` ruby
class SomeJob
  include Disque::Job
  disc queue: "foo"
  disc other_option: "blah"
end

SomeJob.queue #=> "default"
```

Dunno, I feel like we should be more explicit in general about the worker configuration. Not a huge fan of the `disc` method in particular, and even less of chucking more behaviour on it.

Why not just use a different `disque_options` accessor for the hash of options that will be forwarded to Disque?